### ISSUE TITLE
Differing formatting for empty lists with dhall-to-yaml and dhall-to-yaml-ng

### ISSUE THREAD

**Issue by sjakobi** [dhall-lang/dhall-haskell#1560]
```
$ echo '[] : List Bool' | cabal run dhall-to-yaml
 []
$ echo '[] : List Bool' | cabal run dhall-to-yaml-ng
[]
```

Note the leading space with `dhall-to-yaml`.

@patrickmn Could you take a look?

**Comment by olivierdeckers** [comment]
aeson-yaml is outputting a space before the list:
`print $ encode ([] :: [Int])`
outputs " []\n"

**Comment by patrickmn** [comment]
Thank you for the fix, @olivierdeckers!

**Comment by olivierdeckers** [comment]
My pleasure!

**Comment by sjakobi** [comment]
Oh, I hadn't noticed that the fix was released! :)

So we just need to raise the lower bound on `aeson-yaml` to `>= 1.0.4.0`.

May I hope that that can be done as a part of #1561 or should we handle this separately @patrickmn ? :)

**Comment by patrickmn** [comment]
Yeah, both should be fixed in 1.0.5.0.
